package.json

```diff
@@ -70,7 +74,17 @@
     "yup": "^0.32.11"
   },
   "devDependencies": {
+    "@babel/core": "^7.17.9",
```

Comment:
Do we really need `@babel/core` and `babel-loader`? Cause we use TypeScript so theoretically no need for transpiling through another tool but might be required by either Jest or Storybook?

Reply:
yeah, this is correct. I needed them for Storybook, but you're right, TypeScript handles this so no need to have babel here. I will remove them, thanks